libs/payments/stripe/src/lib/stripe.manager.ts

```diff
@@ -21,4 +28,16 @@ export class StripeManager {
       customer.tax?.automatic_tax === 'not_collecting'
     );
   }
+
+  async customerChanged(uid: string, email: string) {
```

Comment:
Should this method be in `StripeManager`? Since it's dependent on a bunch of other services, would `StripeService` be a better place for this?